portfolio/src/main/webapp/index.html

```diff
@@ -6,14 +6,20 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body id="page-background">
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1 class= "font-stylings">My Portfolio</h1>
```

Comment:
remove the spaces between attribute and its value
```suggestion
      <h1 class="font-stylings">My Portfolio</h1>
```